pkg/generators/forwarding/fluentd/rsyslog_conf.go

```diff
@@ -0,0 +1,46 @@
+package fluentd
```

Comment:
I believe this file is obsolete if the remaining bits are opinionated

Reply:
Right, removed